spec/example_app/app/assets/stylesheets/administrate/customize.scss

```diff
@@ -0,0 +1,4 @@
+.custom-administrate-css-hidden {
```

Comment:
Just a thought on this: since we're encouraging people to override this file, maybe it's more of a public API vs. a private API, if that's the division we want the naming scheme to signify. In that case no underscore would be best. If the naming scheme is more about top-level vs. included, then an underscore is appropriate.